src/textual/app.py

```diff
@@ -128,13 +129,54 @@ class CssPathError(Exception):
 
 
 class _NullFile:
+    """A file-like where writes go nowhere."""
+
     def write(self, text: str) -> None:
         pass
 
     def flush(self) -> None:
         pass
 
 
+class _WriterThread(threading.Thread):
```

Comment:
Does the docstring rule hold for methods of a "private" class, in the same way that it holds for private methods?

Reply:
In general, yes! Although this class is a file-like, which already as a well defined and documented interface. Seems superfluous to add docstrings.

Reply:
But you have shamed me in to adding docstrings!